src/Microsoft.Health.Dicom.Core/Features/CustomTag/IIndexableDicomTagService.cs

```diff
@@ -0,0 +1,24 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Health.Dicom.Core.Features.CustomTag
+{
+    /// <summary>
+    /// Service provides indexable dicom tags.
+    /// </summary>
+    public interface IIndexableDicomTagService
```

Comment:
indexTagService? the name is really long

Reply:
sure